digital-twin-model/src/sdv_v2.rs

```diff
@@ -0,0 +1,621 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+// This file contains the generated code for the Software Defined Vehicle (SDV) model.
+// This code is manually generated today, but in the future it should be automatically generated from the DTDL.
+
+/// The context value for all JSON-LD generated by the code in this file.
+fn context() -> Vec<String> {
+    vec!["dtmi:dtdl:context;2".to_string(), "dtmi:sdv:context;2".to_string()]
+}
+
+/// Constants and type defintiions (including JSON serialization/deserialization) for each interface in the SDV model.
+#[allow(dead_code)]
+pub mod airbag_seat_massager {
+    pub const ID: &str = "dtmi:sdv:airbag_seat_massager;1";
+    pub fn id() -> String {
```

Comment:
Some of the macros require a function rather than a literal, e.g. 

                #[derivative(Default(
                    value = "crate::sdv_v2::airbag_seat_massager::store_sequence::request::id()"
                ))]
                
 So it is necessary to provide both a const (ID) and a function (id).

Reply:
Did you try my suggestion? You can put arbitrary rust code in the macro, it doesn't require these functions. The rust playground is back up so I can confirm that my suggestion will work. You can also see this simplified example from the playground which does not need a function definition: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=1a392d55f1959caf4de8d4f9f3fc5d7e

Reply:
Fixed.